tested with and without credentials

pepgid opened the pull request.

Commit:
tested with and without credentials

YaroslavPepticom commented on wielder/util/terraform_credential_helper.py:
```diff
@@ -65,7 +65,18 @@ def get_aws_mfa_cred_command(role_name):
+
+    env_cred = get_aws_mfa_cred(role_name)
 
     as_string_cmd = as_export_cmd(env_cred)
```
Rename something like:credential_to_cmd

pepgid replied:
renamed and renamed module to better reflect the level of abstraction and reusability

YaroslavPepticom commented on wielder/util/terraform_credential_helper.py:
```diff
@@ -79,25 +90,25 @@ def get_aws_mfa_cred_command(role_name):
 if __name__ == "__main__":
```
move commented to test file

pepgid replied:
fixed by moving to test. remember it is useful for cli

Commit:
moved dirty testing for use of terraform credentials to test cleaner but not perfect

Commit:
renaming for better cohesion and readability

Commit:
fixed wielder versions
added botocore to dependencies
moved session to utils
tested with and without config
